### PR TITLE
compatability matrix scrapers should not add new patch versions if no changes

### PR DESCRIPTION
**Pull request by seemywingz**
this PR reduces the noise of patch version compatibility changes and add placeholders and new scrpers

## Checklist

- [ x] If required, I have updated the Plural documentation accordingly.
- [ x] I have added tests to cover my changes.
- [ x] I have added a meaningful title and summary to convey the impact of this PR to a user.

### CODE REVIEW

File: utils/compatibility/scrapers/strimzi-kafka.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    target_tables = find_target_tables(sections)
+    if target_tables.__len__() >= 1:
+        rows = extract_table_data(target_tables)
+        update_compatibility_info(
+            f"../../static/compatibilities/{app_name}.yaml", rows
+        )
+    else:
+        print_error("No compatibility information found.")
+
+    # update_chart_versions(app_name)
```
Comment (michaeljguarino) [inline utils/compatibility/scrapers/strimzi-kafka.py:46]: why are you commenting these out in a lot of the scrapers?

Comment (seemywingz) [reply utils/compatibility/scrapers/strimzi-kafka.py:46]: because they are not implemented yet, they are placeholders

Comment (michaeljguarino) [reply utils/compatibility/scrapers/strimzi-kafka.py:46]: But there's a few that were there like argo-rollouts, ciliium, etc that were there prior.

Comment (seemywingz) [reply utils/compatibility/scrapers/strimzi-kafka.py]: I reenabled them. They were updating the helm chart versions, but some need the actual compatibility scraping configured.

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**